Mergin/validation.py

```diff
@@ -356,3 +372,5 @@ def warning_display_string(warning_id):
         return "Snapping is currently disabled in this QGIS project, it will be thus disabled in Mergin Maps Input"
     elif warning_id == Warning.MERGIN_SNAPPING_NOT_ENABLED:
         return "Snapping is currently enabled in this QGIS project, but not enabled in Mergin Maps Input"
+    elif warning_id == Warning.MISSING_DATUM_SHIFT_GRID:
+        return "Required datum shift grid is missed. <a href='#fix_datum_shift_grids'>Fix.</a>"
```

Comment:
"... is missing. Fix issue."

Reply:
"Fix the issue"

And if there's enough space for more explanation, I would add another sentence, e.g. "Reprojection may not work correctly."